Assignment-1/code/style.css

```diff
@@ -1 +1,69 @@
+.container {
+    width: auto;
+    height: auto;
+}
+.row{
+    display: block;
+    width: auto;
+}
+.yellow{
+    display: flex;
+}
+.red{
+    display: flex;
+}
+.blue{
+    display: flex;
+}
+.yellow .content{
+    margin: 10px;
+    background-color: orange;
+    height: 100px;
+}
+.red .content{
+    margin: 10px;
+    background-color: red;
+    height: 200px;
+}
+.blue .content{
+    margin: 10px;
+    background-color: blue;
+    height: 100px;
+}
+/*----------Min width 1024-------*/
+@media screen and (min-device-width:1024px){
+.yellow .content{
```

Comment:
hey, can u write in a single line because there is a repetition inside this media screen 1024px.